package.json

```diff
@@ -1,5 +1,17 @@
 {
-  "devDependencies": {
+  "name": "style-dictionary-demo",
+  "version": "1.0.0",
```

Comment:
this is very much a nit, but do you think we should start the versioning at 0.0.1 so that when we publish, we publish v1?